examples/step-87/doc/results.dox

```diff
@@ -0,0 +1,269 @@
+<h1>Results</h1>
+
+<h3>Mini example 0</h3>
+
+We present a part of the screen output. It shows, for each point, the
```

Comment:
```suggestion
We present a part of the screen output. It shows, for each point, the
```
screen output could maybe be "log output" or "terminal output". Screen seems a bit vague.

Reply:
I agree, I decided to use "terminal output".